src/courseware/course/sidebar/sidebars/course-outline/CourseOutlineTray.scss

```diff
@@ -1,19 +1,51 @@
-#courseHome-outline {
-    .is-open > .collapsible-trigger {
-        background: #f5f5f5;
+.outline-sidebar:not(.collapsed) {
+    width: 32.125rem;
+    max-width: 100%;
+    overflow: auto;
+}
+
+.outline-sidebar-heading-wrapper {
+    border: 1px solid #d7d3d1;
+
+    .outline-sidebar-heading {
+        font-weight: 700;
     }
+}
+
+.course-outline-sidebar-section {
+    background: $white;
+    border: 1px solid #d7d3d1;
+
+    button {
+        line-height: 1.75rem;
+
+        &.focus::before,
+        &:focus::before {
+            border-radius: 0;
+        }
+    }
+}
+
+.outline-sidebar-toggle-btn {
+    font-size: 1.5rem;
+
+    .collapsed & {
+        transform: scale(-1, 1);
+    }
+}
+
+#outline-sidebar-outline {
+    margin-top: -1px;
 
     li {
         font-size: 1rem;
-        line-height: 1.25rem;
+        line-height: 1.5rem;
 
         .collapsible-trigger {
-            padding: .625rem $spacer;
-            position: relative;
+            padding: 1.375rem 1.5rem 1.375rem 3.375rem;
```

Comment:
[nit]: Please, try to use `$spacers` from Paragon for all paddings and margins

![Screenshot 2024-03-12 at 16 30 18](https://github.com/raccoongang/frontend-app-learning/assets/93188219/aec37199-7c3d-4e43-aae6-ee5185619da0)

Reply:
Changed